hack/kokoro/release.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2021 The Skaffold Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+export GO_VERSION=1.17.3
+
+# Replace go installation with version of our choosing
+rm -Rf /usr/local/go && mkdir /usr/local/go
+curl --fail --show-error --silent --location https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz \
```

Comment:
It would be good to compare the SHA256 values to a known value.

Reply:
done 👍🏼